posts/2020/08/the-ibis-backends.md

```diff
@@ -0,0 +1,359 @@
+      <td>7</td>
+      <td>1</td>
+      <td>2</td>
+      <td>5</td>
+    </tr>
+    <tr>
+      <th>2020</th>
+      <td>38</td>
+      <td>3</td>
+      <td>4</td>
+      <td>2</td>
+      <td>4</td>
+      <td>1</td>
+      <td>2</td>
+      <td>1</td>
+      <td>3</td>
+      <td>4</td>
+      <td>4</td>
+      <td></td>
+    </tr>
+  </tbody>ibis direct execution backends like pandas and hdf5 operate on conventional in-memory python objects. pandas is the gold standard for structured data in python, and inspires the API for ibis.
+
+
+</table>
+</div>
+<br>
+
+> We note an early focus `ibis.sqlite`, `sqlalchemy` and `ibis.impala`. 
+Later, work began on the `pandas` backend rounding out the three different types of backgrounds.
+From this point, improvements were made to these key backends as `ibis.clickhouse`, `ibis.spark` and `ibis.postgres`. 
+For the past 3 years, Quansight, in partnership with OmniSci, added the `ibis.omniscidb`
+string generating backend. Further, our responsibilities have expanded
+to support `ibis` as community maintainers through Quansight Labs. 
+This collaboration introduced geospatial functionality to `ibis` for several backends.
+
+### Currently, there is an ongoing effort to add sqlserver backends
+
+There are on going efforts to introduce [SQL Server][sql-server] support.  
+What other backends would you like to see for `ibis`?
+Maybe our community could benefit from a `dask` backend or `altair` backend?
+
+## `ibis` direct execution backends
+
+`ibis` direct execution backends like `pandas` and `hdf5` operate on conventional in-memory python objects.
+`pandas` is the gold standard for structured data in python, and inspires the API for `ibis`.
+
+
+```python
+    pd = ibis.pandas.connect({'A': pandas.util.testing.makeDataFrame()})
+```
+
+`pd` is an `ibis` backend based off `pandibis direct execution backends like pandas and hdf5 operate on conventional in-memory python objects. pandas is the gold standard for structured data in python, and inspires the API for ibis.
+
+as.DataFrame` objects.
+
+```python
+    expression = pd.table('A').head()
+```
+
+`expression` is an `ibis` query, that has `expression.compile` and `expression.execute` methods.
+We'll recognize the __execute__ method when we return `pandas.DataFrame`s from `ibis` expression.
+The __compile__ method does not trigger any computation, rather it constructs an intermediate form
+that is interpretted by a backend.
+
+```python
+    >>> assert isinstance(expression.compile(), ibis.expr.types.TableExpr)
+```
+
+In the case of direction execution backends, the `expression` compiles to an the original `ibis` 
+expression.  And the computation is carried out based on a set of recipes defined in `ibis`.
+
+In general, we would typically do this work directly in `pandas`, however this work is
+practical in mocking tests for expressions independent of backends.
+
+> Learn more about the [HDF5 direct execution backend in the `ibis` tests][test-hdf5].
```

Comment:
If you want to make it stand out, maybe use italics? The font on `> ....` is too tiny

Reply:
i think the styling on out blockquotes might be off if the text is appearing weird.